apps/cowswap-frontend/src/modules/tenderly/const.ts

```diff
@@ -0,0 +1,6 @@
+const TENDERLY_ORG_NAME = process.env.REACT_APP_TENDERLY_ORG_NAME
+const TENDERLY_PROJECT_NAME = process.env.REACT_APP_TENDERLY_PROJECT_NAME
+
+export const getSimulationLink = (simulationId: string): string => {
+  return `https://dashboard.tenderly.co/${TENDERLY_ORG_NAME}/${TENDERLY_PROJECT_NAME}/simulator/${simulationId}`
+}
```

Comment:
Are those still required? I thought we would get everything from BFF.
Like, return the whole simulation URL from the simulation endpoint.